### PR TITLE
[FIX] delivery_package_type_number_parcels: make number of parcels computed

### PR DESCRIPTION
**Pull request by sbejaoui**
avoid related for this field as we don't want to change the pakcage type value when it changes in the package

cc/ @lmignon

### CODE REVIEW

File: delivery_package_type_number_parcels/models/stock_quant_package.py [hunk 1]
```diff
@@ -1,13 +1,17 @@
...
+from odoo import api, fields, models
 
 
 class StockQuantPackage(models.Model):
     _inherit = "stock.quant.package"
 
     number_of_parcels = fields.Integer(
-        related="package_type_id.number_of_parcels",
-        store=True,
+        compute="_compute_number_of_parcels", store=True, readonly=True
```
Comment (lmignon) [inline delivery_package_type_number_parcels/models/stock_quant_package.py]:
```suggestion
        compute="_compute_number_of_parcels", store=True, readonly=False
```

**Review by lmignon — CHANGES_REQUESTED** [review]
@sbejaoui IMO, the compute should be `readonly=False`

File: delivery_package_type_number_parcels/models/stock_quant_package.py (hunk 1 above)
Comment (sbejaoui) [reply delivery_package_type_number_parcels/models/stock_quant_package.py]: done

**Review by rousseldenis — APPROVED** [review]
LGTM

**Review by lmignon — APPROVED** [review]

**Comment by sbejaoui** [comment]
ready to merge

**Comment by rousseldenis** [comment]
@sbejaoui Could you just add the module name in commit message ? Thanks

**Comment by sbejaoui** [comment]
> @sbejaoui Could you just add the module name in commit message ? Thanks

done

**Pull request merged**